tools/deps.bzl

```diff
@@ -3,10 +3,10 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_r
 
 def bark_dependencies():
     _maybe(
-    git_repository,
-    name = "bark_project",
-    branch = "master",
-    remote = "https://github.com/bark-simulator/bark"
+        git_repository,
+        name = "pybind11_bazel",
+        commit="378ce0523b85ad1748cad49ad9b739a91000e81c",
+        remote = "https://github.com/patrickhart/pybind11_bazel"
```

Comment:
why do you use your fork?

Reply:
the main repo ist not being maintained and no pull requests are accepted. There is an issue that describes how to "fix" the error

Reply:
then let's fork it as bark_simulator?